tests/alignment_test_cases.json

```diff
@@ -243,7 +243,7 @@
     {
         "case_name": "Test alignment: improper clef",
         "text_input": "in diebus eius obsessa est",
-        "vol_input": "-j---1---f---df--f--f---f--efgfe---g--hg-hgf--ef-gef---ed---4",
+        "vol_input": "-j---1--f---df--f--f---f--efgfe---g--hg-hgf--ef-gef---ed---4",
```

Comment:
should this be two test cases? One for `-j---1---f---...` and one for `1--f---...`?

Reply:
probably...will add